session-manager/common/session/Session.cpp

```diff
@@ -304,6 +309,32 @@ namespace ogon { namespace sessionmanager { namespace session {
 			return false;
 		}
 
+		configNS::PropertyManager *propertyManager = APP_CONTEXT.getPropertyManager();
+
+		std::string queryStringDefaultModule = std::string("module");
+
+		if (!propertyManager->getPropertyString(mSessionID, queryStringDefaultModule, defaultmodule)) {
+			defaultmodule = "xsession";
+		}
```

Comment:
It's been some time since I tested this, but as far as I recall, I didn't succeed in adding a new backend by copying existing settings and replacing xsession with something else.
Just didn't work (and I didn't investigate any further).
The above hack "worked for me" and surely needs some cleanups from someone who knows more about how ogon works ;-)

Reply:
wohoo. That was a quick reply. Just to make sure this was a workaround to get the name of the config uses that you can query the settings?

Reply:
Glad to see some activity on this project again!
Yes, I needed to get some settings and I did so by using a hard-coded session string (not knowing how to get to it any other way).